File: test_archives/py_functions/batch4/function5.py
def function5(str1, str2, int1, int2, float1, float2, bool1):
  """This function takes in two strings, two integers, two floats, and a boolean,
  and returns one final value.
  """
  final_value = str1[0] + str2[-1] + str(int1*int2) + str(float1/float2) + str(bool1)
  return final_value

File: test_archives/cpp_functions/batch4/function5.cpp
#include <iostream>
#include <string>
using namespace std;

string function5(string str1, string str2, int int1, int int2, float float1, float float2, bool bool1) {
  string final_value = str1[0] + str2[str2.length()-1] + to_string(int1*int2) + to_string(float1/float2) + to_string(bool1);
  return final_value;
}